packages/visualizations/src/components/Map/MapRender.svelte

```diff
@@ -82,12 +80,11 @@ TODO:
 
     function initializeResizer() {
         // Set a resizeObserver to resize map on container size changes
+        // TODO: Do we really want to reset to the initial bbox each time?
         resizer = new ResizeObserver(
             debounce(() => {
                 map.resize();
-                if (bbox) {
-                    fitMapToBbox(bbox);
-                }
+                fitMapToBbox(bbox);
```

Comment:
I think here we should keep some checks before calling fitMapToBbox as I have this error in the console when using it in Studio only on the first render, it looks like fitMapToBbox is called before having all data.

![Capture d’écran 2022-05-20 à 17 10 43](https://user-images.githubusercontent.com/84501475/169558619-db70067b-cabd-43a3-abf8-cf4151af4a2b.png)

If you also encounter this error, something like this seems to be secure but a check on bbox could be enough:  
```
if (mapReady && bbox) {
     fitMapToBbox(bbox);
}
```